generated/niscope/niscope/niscope_pb2.py

```diff
@@ -15,7 +15,7 @@
ionCoefficientsResponse\x12s\n\x16GetScalingCoefficients\x12+.niscope_grpc.GetScalingCoefficientsRequest\x1a,.niscope_grpc.GetScalingCoefficientsResponse\x12v\n\x17GetStreamEndpointHandle\x12,.niscope_grpc.GetStreamEndpointHandleRequest\x1a-.niscope_grpc.GetStreamEndpointHandleResponse\x12\x97\x01\n\"ImportAttributeConfigurationBuffer\x12\x37.niscope_grpc.ImportAttributeConfigurationBufferRequest\x1a\x38.niscope_grpc.ImportAttributeConfigurationBufferResponse\x12\x91\x01\n ImportAttributeConfigurationFile\x12\x35.niscope_grpc.ImportAttributeConfigurationFileRequest\x1a\x36.niscope_grpc.ImportAttributeConfigurationFileResponse\x12=\n\x04Init\x12\x19.niscope_grpc.InitRequest\x1a\x1a.niscope_grpc.InitResponse\x12^\n\x0fInitWithOptions\x12$.niscope_grpc.InitWithOptionsRequest\x1a%.niscope_grpc.InitWithOptionsResponse\x12j\n\x13InitiateAcquisition\x12(.niscope_grpc.InitiateAcquisitionRequest\x1a).niscope_grpc.InitiateAcquisitionResponse\x12\x85\x01\n\x1cProbeCompensationSignalStart\x12\x31.niscope_grpc.ProbeCompensationSignalStartRequest\x1a\x32.niscope_grpc.ProbeCompensationSignalStartResponse\x12\x82\x01\n\x1bProbeCompensationSignalStop\x12\x30.niscope_grpc.ProbeCompensationSignalStopRequest\x1a\x31.niscope_grpc.ProbeCompensationSignalStopResponse\x12=\n\x04Read\x12\x19.niscope_grpc.ReadRequest\x1a\x1a.niscope_grpc.ReadResponse\x12^\n\x0fReadMeasurement\x12$.niscope_grpc.ReadMeasurementRequest\x1a%.niscope_grpc.ReadMeasurementResponse\x12@\n\x05Reset\x12\x1a.niscope_grpc.ResetRequest\x1a\x1b.niscope_grpc.ResetResponse\x12R\n\x0bResetDevice\x12 .niscope_grpc.ResetDeviceRequest\x1a!.niscope_grpc.ResetDeviceResponse\x12X\n\rRevisionQuery\x12\".niscope_grpc.RevisionQueryRequest\x1a#.niscope_grpc.RevisionQueryResponse\x12O\n\nSampleMode\x12\x1f.niscope_grpc.SampleModeRequest\x1a .niscope_grpc.SampleModeResponse\x12O\n\nSampleRate\x12\x1f.niscope_grpc.SampleRateRequest\x1a .niscope_grpc.SampleRateResponse\x12I\n\x08SelfTest\x12\x1d.niscope_grpc.SelfTestRequest\x1a\x1e.niscope_grpc.SelfTestResponse\x12v\n\x17SendSoftwareTriggerEdge\x12,.niscope_grpc.SendSoftwareTriggerEdgeRequest\x1a-.niscope_grpc.SendSoftwareTriggerEdgeResponse\x12p\n\x15SetAttributeViBoolean\x12*.niscope_grpc.SetAttributeViBooleanRequest\x1a+.niscope_grpc.SetAttributeViBooleanResponse\x12j\n\x13SetAttributeViInt32\x12(.niscope_grpc.SetAttributeViInt32Request\x1a).niscope_grpc.SetAttributeViInt32Response\x12j\n\x13SetAttributeViInt64\x12(.niscope_grpc.SetAttributeViInt64Request\x1a).niscope_grpc.SetAttributeViInt64Response\x12m\n\x14SetAttributeViReal64\x12).niscope_grpc.SetAttributeViReal64Request\x1a*.niscope_grpc.SetAttributeViReal64Response\x12p\n\x15SetAttributeViSession\x12*.niscope_grpc.SetAttributeViSessionRequest\x1a+.niscope_grpc.SetAttributeViSessionResponse\x12m\n\x14SetAttributeViString\x12).niscope_grpc.SetAttributeViStringRequest\x1a*.niscope_grpc.SetAttributeViStringResponseB?\n\x11\x63om.ni.grpc.scopeB\x07NiScopeP\x01\xaa\x02\x1eNationalInstruments.Grpc.Scopeb\x06proto3')
```

Comment:
The changes to this file appear significant.
Should we be concerned?

Reply:
I don't think you should worry in this case.

This byte array is a serialized description of the `niscope.proto` file. Adding a field to a message inserted some bytes in the middle, shifted the end of each line to the beginning of the next line, and increased all of the offsets listed below.

Using a different diff tool such as Beyond Compare shows this more clearly, but it's still hard to verify that most of the byte array didn't change.
![image](https://user-images.githubusercontent.com/17914399/231295424-ea893ce3-1da0-4987-bcb8-5c153a01ad08.png)

You should worry when updating a `.proto` file that is duplicated between multiple Python packages, such as `session.proto`. If `session.proto`'s descriptor doesn't match between nimi-python, measurementlink-python, and nidaqmx-python, users will get errors when they load more than one of these packages. We have technical debt AB#2250775 about this.